pkgs/development/libraries/libcaca/default.nix

```diff
@@ -11,9 +11,9 @@ stdenv.mkDerivation rec {
     sha256 = "1x3j6yfyxl52adgnabycr0n38j9hx2j74la0hz0n8cnh9ry4d2qj";
   };
 
-  configureFlags = "--disable-x11 --disable-imlib2 --disable-doc";
```

Comment:
I don't know why it was there in the first place.
The X11 was wanted by the compiler after I added imlib2.
The pkg also compiles without --disable-x11.